app/src/main/java/com/example/geoquiz/QuizActivity.kt

```diff
@@ -80,27 +68,19 @@ class QuizActivity : AppCompatActivity() {
         outState.putBooleanArray(QUESTIONS_ANSWERED_KEY, quizViewModel.questionWasAnswered)
     }
 
-    override fun onStop() {
-        super.onStop()
-        Log.d(TAG, "onStop() called")
-    }
-
-    override fun onDestroy() {
-        super.onDestroy()
-        Log.d(TAG, "onDestroy() called")
-    }
-
     private fun restoreState(savedInstanceState: Bundle?) {
-        if (savedInstanceState != null) {
+        savedInstanceState?.let {
```

Comment:
It's hard to follow why the if has been changed to let. If you do not benefit from having a non null value provided by let, I prefer to use the if not null clause. It's easier to understand because this way, I don't know where the `it` from line 76, comes from, I have to really look closely at the code to understand

Reply:
If you want to use the let, name the param inside `let` to bundle or something and use it instead of `savedInstanceState `